### ISSUE TITLE
How to change navigatorStyle dynamically?

### ISSUE THREAD

**Issue by henrikra** [wix/react-native-navigation#985]
How can I change `navigatorStyle` property for example `navBarBackgroundColor` dynamically at run time? For example in my app I want to change navigator background when user goes online :)

Only solution I have figured so far to hack this is to use `resetTo` without animation. This is otherwise good but it rerenders the view  :S

**Comment by Ehesp** [comment]
Not sure how to change it dynamically, but you could set it to transparent and have a View hooked up behind it which changes color?

**Comment by Zcgong** [comment]
+1

**Comment by guyca** [comment]
You can use `setStyle`, available in 1.1.42
```js
this.props.navigator.setStyle({
  screenBackgroundColor: 'red',
  navBarButtonColor: 'blue'
});
```
